[2025-03-07T11:03:00.735+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2025-03-04T00:00:00+00:00 [queued]>
[2025-03-07T11:03:00.744+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2025-03-04T00:00:00+00:00 [queued]>
[2025-03-07T11:03:00.744+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-03-07T11:03:00.762+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2025-03-04 00:00:00+00:00
[2025-03-07T11:03:00.772+0000] {standard_task_runner.py:57} INFO - Started process 1051 to run task
[2025-03-07T11:03:00.774+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2025-03-04T00:00:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/postgres_operator.py', '--cfg-path', '/tmp/tmp_2ytktqf']
[2025-03-07T11:03:00.777+0000] {standard_task_runner.py:85} INFO - Job 42: Subtask insert_into_table
[2025-03-07T11:03:00.829+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2025-03-04T00:00:00+00:00 [running]> on host 5588030d4cfa
[2025-03-07T11:03:00.914+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-04T00:00:00+00:00'
[2025-03-07T11:03:00.915+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2025-03-04', 'dag_with_postgres_operator_v03')
        
[2025-03-07T11:03:00.921+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T11:03:00.930+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T11:03:00.934+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2025-03-04', 'dag_with_postgres_operator_v03')
        , parameters: None
[2025-03-07T11:03:00.935+0000] {sql.py:427} INFO - Rows affected: 1
[2025-03-07T11:03:00.947+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20250304T000000, start_date=20250307T110300, end_date=20250307T110300
[2025-03-07T11:03:00.987+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-07T11:03:01.004+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-07T11:32:47.207+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2025-03-04T00:00:00+00:00 [queued]>
[2025-03-07T11:32:47.213+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2025-03-04T00:00:00+00:00 [queued]>
[2025-03-07T11:32:47.213+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-03-07T11:32:47.224+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2025-03-04 00:00:00+00:00
[2025-03-07T11:32:47.230+0000] {standard_task_runner.py:57} INFO - Started process 874 to run task
[2025-03-07T11:32:47.233+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2025-03-04T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/postgres_operator.py', '--cfg-path', '/tmp/tmpfc05nlrd']
[2025-03-07T11:32:47.236+0000] {standard_task_runner.py:85} INFO - Job 28: Subtask insert_into_table
[2025-03-07T11:32:47.282+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2025-03-04T00:00:00+00:00 [running]> on host 5837dac536a6
[2025-03-07T11:32:47.368+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-04T00:00:00+00:00'
[2025-03-07T11:32:47.368+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2025-03-04', 'dag_with_postgres_operator_v03')
        
[2025-03-07T11:32:47.377+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T11:32:47.386+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T11:32:47.390+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2025-03-04', 'dag_with_postgres_operator_v03')
        , parameters: None
[2025-03-07T11:32:47.393+0000] {sql.py:427} INFO - Rows affected: 1
[2025-03-07T11:32:47.401+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20250304T000000, start_date=20250307T113247, end_date=20250307T113247
[2025-03-07T11:32:47.445+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-07T11:32:47.459+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-07T12:05:07.943+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2025-03-04T00:00:00+00:00 [queued]>
[2025-03-07T12:05:07.949+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2025-03-04T00:00:00+00:00 [queued]>
[2025-03-07T12:05:07.949+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-03-07T12:05:07.959+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2025-03-04 00:00:00+00:00
[2025-03-07T12:05:07.965+0000] {standard_task_runner.py:57} INFO - Started process 490 to run task
[2025-03-07T12:05:07.968+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v03', 'insert_into_table', 'scheduled__2025-03-04T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/postgres_operator.py', '--cfg-path', '/tmp/tmpg42hzh8p']
[2025-03-07T12:05:07.970+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask insert_into_table
[2025-03-07T12:05:08.014+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_with_postgres_operator_v03.insert_into_table scheduled__2025-03-04T00:00:00+00:00 [running]> on host afc6f8faa09c
[2025-03-07T12:05:08.079+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v03' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-04T00:00:00+00:00'
[2025-03-07T12:05:08.080+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2025-03-04', 'dag_with_postgres_operator_v03')
        
[2025-03-07T12:05:08.087+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T12:05:08.096+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2025-03-07T12:05:08.099+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2025-03-04', 'dag_with_postgres_operator_v03')
        , parameters: None
[2025-03-07T12:05:08.102+0000] {sql.py:427} INFO - Rows affected: 1
[2025-03-07T12:05:08.109+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v03, task_id=insert_into_table, execution_date=20250304T000000, start_date=20250307T120507, end_date=20250307T120508
[2025-03-07T12:05:08.139+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-07T12:05:08.151+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
